resources/function-controller/templates/cluster-role.yaml

```diff
@@ -8,12 +8,9 @@ rules:
 - apiGroups:
```

Comment:
make sure to copy those generated rbac etc after review from everyone is done

Reply:
Yup, there are copied already to charts, but I will double-check before merge...